### PR TITLE
feat: add README and imports in index.ts

### PR DESCRIPTION
**Pull request by EnkiP**
clickup #86bx306pm

### CODE REVIEW

**Comment by ghusse** [comment]
Task linked: [CU-86bx306pm Story #3.4 - Users want to bootstrap a CC project scaffolding from forest-cli - 5 pts](https://app.clickup.com/t/86bx306pm)

File: README.md [hunk 1]
```diff
@@ -1 +1,43 @@
-Clone this repo to start customizing your Forest Admin in the cloud
+# Usage
+
+Once you created a new (cloud) project on forest admin,
+you can use this package to add code customizations to it.
+
+The workflow to add code customizations is the following:
+
+1. [Bootstrap the code customization template](#bootstrap-the-code-customization-template), to be able to write your customizations.
+2. [Generate typings](#generate-typings) for typescript to know your project types.
```
Comment (realSpok) [inline README.md:9]: This is already included in Boostrap step

File: README.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+
+and install the dependencies
+
+`yarn` or `npm install`
+
+# Generate typings
+
+`yarn update-typings` or `npm run update-typings`
+
+This command will request you to login to your project.
```
Comment (realSpok) [inline README.md:29]: Bootstrap also requires you to log in

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
...
+It will then generate typings according to the structure of your
+database and your current code customizations.
```
Comment (realSpok) [inline README.md:31]: The typings are here to provide autocompletion in your IDE to help you write faster and less error prone code

File: README.md [hunk 4]
```diff
@@ -1 +1,43 @@
...
+
+You can execute this command on a regular basis, to keep your linter
```
Comment (realSpok) [inline README.md:33]:
```suggestion
You should execute this command on a regular basis, to keep your IDE
```

File: src/index.ts [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
+  // For instance, you can code custom actions, charts,
+  //   create new fields or relationships, load plugins.
   // refer to the documentation for more details: https://docs.forestadmin.com/developer-guide-agents-nodejs/agent-customization/agent-customization
   //
   // Here is some code examples to get your started
-  /*
+  
   agent.customizeCollection("user", (userCollection) => {
     userCollection.addAction("Anonymize", {
       scope: "Bulk", // This action can be triggered product on one or more products
```
Comment (realSpok) [inline src/index.ts:19]: This action can be triggered on one or multiple users

**Review by Scra3 — APPROVED** [review]

**Pull request merged**